verl/trainer/ppo/ray_trainer.py

```diff
@@ -406,16 +407,32 @@ def _create_dataloader(self):
             'truncation', 'error'
         ), f'dataset truncation {self.train_dataset.truncation} must be the same as config {self.config.data.get("truncation", "error")}'
         # use sampler for better ckpt resume
-        if self.config.data.shuffle:
+        if self.config.data.get('use_dynamic_sampler', False):
```

Comment:
do you think it's possible to rewrite the interface similar to https://github.com/volcengine/verl/pull/832/files, where we can load a custom sampler class, with 2 arugments "data_source" and "data.config", so users have maximum flexibility in extending their own sampler?